### ISSUE TITLE
Namespacing Classes

### ISSUE THREAD

**Issue by HendrikN** [phayes/geoPHP#49]
I'd like to integrate the GeoPHP project into our companies CMS, there is however a problem with the classnames. For example in some client-installations there's already a `Point` object defined. This would be solved by namespacing the Classes (e.g. `geoPHP_Geometry_Point`). 

I absolutely realise that adding namespacing breaks backwards compatibility, but I do like to know what your opinion is regarding this subject.

If this is not the place to start this discussion, feel free to disregard and/or remove this issue!

**Comment by Jelle-S** [comment]
Last comment was 2012. I think it's safe to say that namespacing the classes wouldn't hurt anymore (even on shared hosting).

**Comment by BathoryPeter** [comment]
A pull request with namespaced classes is just waiting to merge: #125

**Comment by gholol** [comment]
@BathoryPeter I looked at your implementation and it's wrong.

> use geoPHP\geoPHP;
> use geoPHP\Geometry\Geometry;
> use geoPHP\Geometry\GeometryCollection;

 You should have a fully qualified class name that should have a top-level namespace name, aka "vendor namespace". You also  MUST use CamelCase.

So the proper way to implement that would look like this:

> use Phayser\GeoPHP\GeoPHP;
> use Phayser\GeoPHP\Geometry\Geometry;
> use Phayser\GeoPHP\Geometry\GeometryCollection;

Implemented in #130

**Comment by BathoryPeter** [comment]
I don't think it is wrong rather an open question. Patrick has to decide about the naming and he / anybody (or me) can refactor it in less then a minute. I'ts just two click in PhpStorm.
